content-cache/src/nginx_manager.py

```diff
@@ -172,17 +225,19 @@ def _load_config() -> None:  # pragma: no cover
     if _systemctl_status_check():
         logger.info("Loading nginx configuration files")
         # This is reload the configuration files without interrupting service.
-        execute_command(["sudo", "nginx", "-s", "reload"])
+        execute_command(["sudo", NGINX_BIN, "-s", "reload"])
```

Comment:
Good catch. Should not have hard coded these values.

Reply:
It's less useful now that we decided to stick to nginx and not switch to openresty. But it was convenient during the tests and if we plan to switch in the future ;)